packages/pg/lib/connection-parameters.js

```diff
@@ -7,6 +7,10 @@ const defaults = require('./defaults')
 const parse = require('pg-connection-string').parse // parses a connection string
 
 const val = function (key, config, envVar) {
+  if (config[key] !== undefined) {
```

Comment:
Original expression used truthiness:

```suggestion
  if (config[key]) {
```